packages/@ember/-internals/metal/lib/computed.ts

```diff
@@ -675,6 +676,25 @@ export class ComputedProperty extends ComputedDescriptor {
     let ret;
 
     try {
+      if (obj instanceof Ember.Component) {
```

Comment:
I think we should avoid the `instanceof` check here. We don't want `@ember/-internals/metal` to have imports from `@ember/-internals/glimmer`, and we can't leverage the `Ember` global (this is partially why the Node tests are failing ATM).

I think we should update to use this flag that we set (and AFAICT is only set for Ember.Component):

https://github.com/emberjs/ember.js/blob/730c638eb52791709cd004ad4324b51ce2b43353/packages/%40ember/-internals/glimmer/lib/component.ts#L718

Which would change things to:

```suggestion
      if (obj.isComponent === true) {
```